projects/fsastorefrontlib/src/cms-components/checkout/components/legal/legal-checkboxes/legal-checkboxes.component.html

```diff
@@ -31,18 +31,17 @@
         </button></ng-container
       >
     </div>
-    <div class="col-12 col-md-6">
-      <ng-container *ngIf="nextCheckoutStep$ | async as nextCheckoutStep"
-        ><button
-          [disabled]="!form.valid || !form.dirty"
```

Comment:
No need for checking `dirty` state of form, it is enough only to check `valid` state.